### ISSUE TITLE
Implement customer satisfaction score measurement

### ISSUE THREAD

**Issue by strogonoff** [riboseinc/jekyll-theme-open-project#45]
We’d want to enable Open Project sites to collect satisfaction score by asking simple questions such as “Did you find what you were looking for?” e.g. at the end of documentation pages.

Recording and aggregation of satisfaction scores and associated comments can be implemented via e.g. AWS DynamoDB, optionally with Lambda. Amplify / AWS Mobile library might simplify using AWS APIs from the browser.

**Comment by ronaldtse** [comment]
Agree. Perhaps it's easier to just use Google Forms to do it.

**Comment by strogonoff** [comment]
Absolutely, I’ll look into integrating Google Forms into the page if it’s simple enough and possible to do without adding friction to the end user (affects measurement).
